docs/code-standards-policy.rst

```diff
@@ -36,6 +43,85 @@ This Policy describes how the Ansible roles and playbooks should be written,
 what ways can be used to combine two or more roles together and how the roles
 should be documented.
 
+
+.. _debops_policy__ref_code_standards_summary:
+
+Summary
+-------
+
+Here's a basic set of principles to be aware while writing roles:
+
+- YAML files MUST use 2 space indents and end with ``.yml``.
+
+- Use spaces around Jinja variable syntax as in ``{{ var }}``.
+
+- When possible use 80 characters line width.
+
+
+**Ansible role: defaults**
+
+- Follow the :ref:`variable naming conventions <debops_policy__ref_code_standards_default_variable_naming_convention>`.
+
+- Make :ref:`conditional code configurable <debops_policy__ref_code_standards_task_conditions>`
+  via default variables.
+
+- :ref:`Comment and structure <debops_policy__ref_code_standards_default_variable_documentation>`
+  default variables with reStructuredText.
+
+- Roles providing features for other roles to use MUST do so by offering
+  dedicated :ref:`dependent variables <debops_policy__ref_code_standards_dependent_variables>`.
+
+- Use `inventory level scoped variables <debops_policy__ref_code_standards_inventory_level_scoped_variables>`
+  to offer mergable configuration keys in multiple inventory scopes.
+
+
+**Ansible role: tasks**
+
+- Make sure the task execution is idempotent.
```

Comment:
I wouldn't want to overdo it, especially that newer Ansible versions (2.2+, I think) have a variable that indicates if Ansible is run in the check mode. Furthermore, not all tasks can be fixed to work correctly in check mode, like script execution, so I would just suggest for this feature to work correctly. Keep in mind that people usually have development environments where roles can be checked properly as well.